pkg/lib/cockpit-components-dropdown.tsx

```diff
@@ -67,12 +73,3 @@ export const KebabDropdown = ({ dropdownItems, position = "end", isDisabled = fa
         </Dropdown>
     );
 };
-
-KebabDropdown.propTypes = {
```

Comment:
Yay, that works for ... typescript but this is also something which happens runtime in React which we now lose. (And this only works for full typescript projects, which more and more seems like a mistake to port projects to typescript).

Reply:
I dropped propTypes annotations on previous ports of code to typescript.  I'm open to leaving them.

Reply:
I know and might have even approved it, but it sounds sensible to keep it around for third party consumers and podman/machines. Does it hurt us to keep it around?

Reply:
#20652 gets to the crux of the issue here, I think.  Maybe we should discuss this there. 

I think we want to be checking our JavaScript with tsc, but not in strict mode.  In that case, -machines and friends would indeed benefit from the types.

Reply:
Only if you use typescript, which we don't require. So proptypes can stay useful. I don't think I rely on them as they are runtime and a warning. But when I do spot such issues I usually fix it.

Reply:
I can bring it back.